queuebot/cogs/queue/__init__.py

```diff
@@ -284,7 +291,12 @@ def has_emoji_slots(self, guild: discord.Guild) -> bool:
                             name=emoji_name, image=await resp.read(), reason='temp blob for vs'
                         ))
 
-            emote_sequence = " \N{SQUARED VS} ".join(map(str, temp_emotes))
+            if self.config.verbose_vs:
+                mapped_lines = [NUMERIC_EMOTES[temp_emotes.index(e)] + str(e) for e in temp_emotes]
+                emote_sequence = VERBOSE_VS_JOINER.join(mapped_lines)
+            else:
+                mapped_lines = [str(e) for e in temp_emotes]
```

Comment:
`mapped_lines` isn't used anywhere else, so you should just use the old format for this condition.